submissions/kazeemErinfolami/database/init.sql

```diff
@@ -0,0 +1,199 @@
+
+
+/*CATEGORIES....*/
+CREATE TABLE categories (
+	id INT NOT NULL AUTO_INCREMENT,
+    category VARCHAR(200) NOT NULL,
+	PRIMARY KEY (id)
+);
+
+INSERT INTO categories (category)
+VALUES ("Shoes"),
+("Men Collection"),
+("Women Collection"),
+("Smart Home"),
+("Smart Phones"),
+("Computers"),
+("Electronics"),
+("Cars"),
+("Vegetable");
+
+SELECT * FROM categories;
+
+
+/*CUSTOMERS...*/
+CREATE TABLE customers (
+	id INT NOT NULL AUTO_INCREMENT,
+	first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+	email VARCHAR(150) NOT NULL,
+	password VARCHAR(40) NOT NULL,
+	date_created DATETIME,
+	PRIMARY KEY (id)
+);
+
+INSERT INTO customers (first_name, last_name, email, password, date_created)
+VALUES ("Tunde", "Ibrahim", "tundeibrahim@gmail.com", "hhsgsty78", now()),
+("Kunle", "Fold", "kunlefold@gmail.com", "8839wkwui", now()),
+("Page", "Lolade","loladepage@yahoo.com", "927267tet", now()),
+("Mary", "Ebube","maryebube@gmail.com", "kdidodhdh737", now()),
+("Joy", "Musa", "joymusa@gmail.com", "hhht67w87", now()),
+("Akim", "Folarin", "folarinakim@gmail.com", "8839wkwui", now()),
+("Muma", "Tonna","tonnamuma@yahoo.com", "927267tet", now()),
+("Edet", "Charles","charlesedet@gmail.com", "kdidodhdh737", now());
+
+SELECT * FROM customers;
+
+
+/*CUSTOMER CONTACTS*/
+CREATE TABLE customers_contacts (
+	id INT NOT NULL AUTO_INCREMENT,
+    customer_id INT NOT NULL,
+	street_address VARCHAR(100) NOT NULL,
+    city VARCHAR(30) NOT NULL,
+	state VARCHAR(15) NOT NULL,
+	zipcode VARCHAR(40),
+	country VARCHAR(15) NOT NULL,
+    phone_number VARCHAR(20) NOT NULL,
+	PRIMARY KEY (id),
+    FOREIGN KEY(customer_id) REFERENCES customers(id)
+);
+
+INSERT INTO customers_contacts (customer_id, street_address, city, state, zipcode, country, phone_number)
+VALUES (1, "House 4 Fishland Avenue", "Lagos", "Lagos", 90282, "Nigeria", 08078726353),
+(2, "Home Land", "CA", "California", 72829, "USA", 896765443),
+(3, "Brown Street", "Munich", "Munich", 78265, "Germany", 07087826756),
+(4, "Street 3A Running Bones", "Yaba", "Ibadun", "", "Nigeria", 09189367252),
+(5, "President Villa Estate", "Ikeja", "Lagos", "", "Nigeria", 09089276739),
+(6, "No 90 China Town", "Ching-Shin", "Seoul", 728282, "Korea", 9975545422),
+(7, "Ikaja VRA", "OSA", "Kogi", "", "Nigeria", 07078783902),
+(8, "Ikota Lekki phase1", "Lekki", "Lagos", 892786, "Nigeria", 09082618725);
+
+SELECT * FROM customers_contacts;
+
+
+/*ORDERS...*/
+CREATE TABLE orders (
+	id INT NOT NULL AUTO_INCREMENT,
+    customer_id INT NOT NULL,
+	order_amount DECIMAL(9, 2) NOT NULL,
+    time_of_order DATETIME NOT NULL,
+	PRIMARY KEY (id),
+    FOREIGN KEY(customer_id) REFERENCES customers(id)
+);
+
+INSERT INTO orders (customer_id, order_amount, time_of_order)
+VALUE (4, 300.00, now()),
+(8, 343.00, now()),
+(2, 800.00, now()),
+(6, 20.00, now()),
+(7, 600.00, now()),
+(1, 390.00, now()),
+(5, 700.00, now()),
+(3, 990.00, now());
+
+SELECT * FROM orders;
+
+
+/*PRODUCTS...*/
+CREATE TABLE products (
```

Comment:
Kindly rework this products table again. Some column(s) is/are not supposed to be part of this table